{% extends 'base.html.twig' %}

{% block title %}Errore interno (500){% endblock %}

{% block content %}
<div class="container py-5">
  <div class="row justify-content-center">
    <div class="col-lg-8 text-center">
      <h1 class="display-4 mb-4">Errore interno</h1>
      <p class="lead mb-4">Si è verificato un errore inatteso. Il problema è stato registrato.</p>
      {% if displayErrorDetails and exception is defined %}
        <div class="text-start alert alert-warning mt-4">
          <h5>Dettagli (solo in modalità debug)</h5>
          <pre class="small mb-0">{{ exception.message }}</pre>
        </div>
      {% endif %}
      <a href="/" class="btn btn-primary mt-3">Torna alla Home</a>
    </div>
  </div>
</div>
{% endblock %}